apps/bundle_deploy/bundle_static.c

```diff
@@ -86,5 +93,30 @@ TVM_DLL void tvm_runtime_get_output(void* runtime, int32_t index, DLTensor* tens
 
 void __attribute__((noreturn)) TVMPlatformAbort(int error_code) {
   fprintf(stderr, "TVMPlatformAbort: %d\n", error_code);
+  void* trace[200];
+  int nptrs = backtrace(trace, sizeof(trace) / sizeof(void*));
+  fprintf(stderr, "backtrace: %d\n", nptrs);
+  if (nptrs < 0) {
+    perror("backtracing");
+  } else {
+    backtrace_symbols_fd(trace, nptrs, STDOUT_FILENO);
```

Comment:
This is a great feature to have backtrace enabled, however, since these functions are not part of the standard c programming language, and it would have potential portability issue, can we add a macro to disable this feature for the platforms where backtracing is not supported?

Reply:
i'm actually just trying to use this to debug the regression, I couldn't reproduce the gpu failure on my side. but, now i've found a node where i should be able to run tvmai/ci-gpu, so i'm going to take that approach.

for merging, I don't know--I know we had issues with the CRT being flaky in the regression before. it might be worth it to leave this code in, since flaky test failures are almost always solved with more data. so, i'd be happy to put it behind a `#define` and enable that for the regression, if you're also happy with that?

Reply:
Yes, this is exactly what I wanted from the previous comment, and I think leaving backtracing flag on should be fine.

Reply:
okay, added a Makefile flag to control this.